tests/test_celery_unique.py

```diff
@@ -331,7 +327,7 @@ def test_creates_new_task_record_when_extant_task_exists(self):
         )
         self.assertIsInstance(rs, AsyncResult)
         self.assertNotEqual(self.redis_client.get(self.test_unique_redis_key), test_extant_tracked_task_id)
-        self.assertEqual(self.redis_client.get(self.test_unique_redis_key), rs.task_id)
+        self.assertEqual(self.redis_client.get(self.test_unique_redis_key), rs.task_id.encode())
```

Comment:
We need `.encode()` here because the returned value is of type `bytes` [python3 only].